modules/async-icq/Makefile

```diff
@@ -2,19 +2,17 @@
 
 PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
 PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
-CHANGED_GO_FILES := $(shell git diff --name-only | grep .go$$ | grep -v pb.go)
-ALL_GO_FILES := $(shell find . -regex ".*\.go$$" | grep -v pb.go)
 VERSION := $(shell echo $(shell git describe --always) | sed 's/^v//')
 COMMIT := $(shell git log -1 --format='%H')
+GOMOD := $(shell go list -m)
 LEDGER_ENABLED ?= true
 BINDIR ?= $(GOPATH)/bin
 BUILDDIR ?= $(CURDIR)/build
 SIMAPP = ./testing/simapp
-MOCKS_DIR = $(CURDIR)/tests/mocks
+MOCKS_DIR = $(CURDIR)/test/mock
```

Comment:
do we still need this? i don't think we use mocks in async-icq anymore

Reply:
b3ad311fbf30ee7473966074b0c2e30156eb4280